python/ray/tests/conftest.py

```diff
@@ -101,6 +101,8 @@ def ray_start_regular_shared(request):
     }])
 def ray_start_shared_local_modes(request):
     param = getattr(request, "param", {})
+    if param["local_mode"] is False and os.name == "nt":
```

Comment:
The tests that use this were failing. I am not sure how to differentiated between multi-node and `local_mode == False`. This skip was needed to avoid hitting the problematic serialization code in `_raylet.pyx`, which caused test failures.